src/app/admin/collectionCentresTable/CollectionCentresTable.tsx

```diff
@@ -0,0 +1,107 @@
+"use client";
+
+import React, { useState } from "react";
+import Table, { TableHeaders } from "@/components/Tables/Table";
+import styled from "styled-components";
+import Modal from "@/components/Modal/Modal";
+import Button from "@mui/material/Button";
+import DeleteIcon from "@mui/icons-material/Delete";
+import RefreshPageButton from "@/app/admin/RefreshPageButton";
+import { Schema } from "@/database_utils";
+import supabase from "@/supabaseClient";
+
+const DangerDialog = styled(Modal)`
+    & #deleteCollectionCentreDialog {
```

Comment:
you shouldn't reference ID at all - you should just use DangerDialog with a background colour, it shouldn't depend on the ID!

Reply:
I have attached a className `header` to the Modal Header so I can target the div using the className instead - let me know if you think it is an acceptable implementation!